package.json

```diff
@@ -21,12 +21,20 @@
     "babel": "^6.5.2",
     "babel-core": "^6.14.0",
     "babel-loader": "^6.2.5",
+    "babel-preset-airbnb": "^2.2.3",
     "babel-preset-es2015": "^6.14.0",
     "babel-preset-react": "^6.11.1",
+    "babel-preset-stage-0": "^6.24.1",
+    "chai": "^3.5.0",
     "css-loader": "^0.25.0",
+    "enzyme": "^2.8.2",
     "file-loader": "^0.9.0",
+    "jsdom": "^10.1.0",
```

Comment:
you will probably need to run an npm update on react and react dom for it to be compatible with this version of jsdom